components/director/internal/domain/runtime/service.go

```diff
@@ -261,6 +258,14 @@ func (s *service) CreateWithMandatoryLabels(ctx context.Context, in model.Runtim
 		in.Labels[key] = value
 	}
 
+	if scenarios, areScenariosInLabels := in.Labels[model.ScenariosKey]; areScenariosInLabels {
+		if err := s.assignRuntimeScenarios(ctx, rtmTenant, id, scenarios); err != nil {
+			return err
+		}
+
+		delete(in.Labels, model.ScenariosKey)
+	}
```

Comment:
I assign them here because on the next line the input labels are upserted and we want to use the Formation API for the "scenarios" label modifications. Maybe we can assign them below(on line 296) with the labels that are coming from ASAs, but at least we need to store the scenarios that are coming from the input in a slice and delete them from the input labels so they will not end up being upserted.

Reply:
Stani has a similar comment, so I will merge this code with the code below.

Reply:
After we discussed it, we can't assign all of the formations at once. The input formations must be assigned in the subaccount and the ASAs formations must be assigned in the global account.